src/FunctionApps/python/FhirServerExport/__init__.py

```diff
@@ -8,19 +8,27 @@
 
 
 def main(req: func.HttpRequest) -> func.HttpResponse:
+    """
+    Main function that activates the FHIR export function app utility.
+    An HTTP request is used to kick off the procedure. For more information,
+    see the README file accompanying the FhirServerExport function app.
+    :param func.HttpRequest req: The request initiating a FHIR export.
+    :return: The response from the FHIR server
+    :rtype: func.HttpResponse
```

Comment:
Actually I misunderstood. Sphinx definitely pulls the types from the function signature, so it's not necessary to document the types in our comments. However, the [config option](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_typehints) `autodoc_typehints` allows us to configure whether the docs show the types of parameters in the function signature, in the parameter descriptions, or both. Does anyone have a preference? The default is in the signature, only

Reply:
My vote is signature only - I can't think of a reason we'd need to double-document.

Reply:
My vote is for parameters only. I think it makes the signature cleaner and moves the types to be next to where the description of the parameter is. Happy to go with majority rule here though. FWIW, pandas places the type with the parameter ([source](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.groupby.html)).

Reply:
I concur have worked on the phone number PR and now the schema code since we started documenting like this I'm getting tired of so much redundant documentation. So if our doc generation tool doesn't need it I would like to stop.

Reply:
I just looked w/ Bryan. We confirmed that Sphinx can get both the parameter and return types from the function signature, so there's no need to include them in the docstring. We also looked at both config options for the output and agreed that it looks nicer to show the types next to the parameter descriptions rather than in the function signature, so I'm going to set that config option as long as there's no one against! And I will go through and remove the redundant types from the docstrings.

Reply:
Sorry I misunderstood the question and thought we were asking where to document type _in our code_.

I don't have a strong preference where to tell Sphinx to put it in the html output, but would lean towards parameters only as Bryan suggested.